traffic_ops/testing/api/v14/divisions_test.go

```diff
@@ -16,19 +16,42 @@ package v14
 */
 
 import (
+	"strings"
 	"testing"
 
 	"github.com/apache/trafficcontrol/lib/go-log"
 	"github.com/apache/trafficcontrol/lib/go-tc"
 )
 
 func TestDivisions(t *testing.T) {
-	WithObjs(t, []TCObj{Parameters, Divisions}, func() {
+	WithObjs(t, []TCObj{Parameters, Divisions, Regions}, func() {
+		TryToDeleteDivision(t)
 		UpdateTestDivisions(t)
 		GetTestDivisions(t)
 	})
 }
 
+func TryToDeleteDivision(t *testing.T) {
+	division := testData.Divisions[0]
+	resp, _, err := TOSession.DeleteDivisionByName(division.Name)
```

Comment:
Doesn't compile, `resp` isn't used, needs to be `_, _, err := TOSession.DeleteDivisionByName(division.Name)`